_data/mu-features.yml

```diff
@@ -1,12 +1,12 @@
 content:
     - title: HTTP
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: REST based microservices, where server-side data are made available through representations of data, typically in JSON or XML format.
```

Comment:
```suggestion
      description: REST-based typed microservices with auto-derived client and server implementations.
```

Reply:
Suggested wording tries to describe more unique features of the library (I only know about the upcoming Scala implementation however, not sure if this applies to Kotlin.)